### DISCUSSION TITLE
Calculating time to disinfect

### DISCUSSION THREAD

**Discussion by Ricco1973** [iwishiwasaneagle/boxofhope#106] [Q&A] [answered]
How are you calculating the amount of time needed to disinfect, and are you basing it on the wattage of the UV light source?  If so what is your equation?

**Comment by iwishiwasaneagle** [comment, marked as answer]
Hi Ricco, 

This is something that we've left for the user to implement. We are not reccommending any hardware, but rather just providing the raw tools. For example, the current schematic assumes you are using a single lower LED [here](https://github.com/iwishiwasaneagle/boxofhope/wiki/Setup---Hardware#wiring-diagram). Obviously this is not the case but the software logic remains the same. 

So in short, we cannot assume that a user will have a specific piece of hardware. However, we are hoping to include a better guide in the future on what hardware to pick.

Hope that answers your question a bit more.

- Jan

**Comment by Ricco1973** [comment]
Since this project is intended for people to build disinfection boxes, I
think it would be appropriate to offer at least a basic suggestion of
disinfection rates.
Or have a calculator that they could plug the information into for the UV
light they've selected. Otherwise people might assume I had it in the
disinfection box
for 5 seconds and now all virus/bacteria is dead  which would not likely be
the case. And is what a lot of off brand manufactures are doing.

**Reply by iwishiwasaneagle** [reply]
Yes absolutely agreed. For now it was important to get a prototype/demonstration together for university so this was overlooked.
